### ISSUE TITLE
[Dropdown] Narrator is not announcing "Collapsed"/"Expanded" when Dropdown is collapsed/expanded 

### ISSUE THREAD

**Issue by Lijing29** [OfficeDev/office-ui-fabric-react#11367]
### Environment Information

- **Package version(s)**: office-ui-fabric-react@7.67.0
- **Browser and OS versions**: Narrator + Edge

### Describe the issue:
Narrator would not announce 'collapsed' or 'expanded' when the dropdown is collapsed or expanded.

### Please provide a reproduction of the issue in a codepen:
https://codepen.io/lijinglin29/pen/gObbvpg

#### Actual behavior:

Narrator does not read the State(Expand/Collapse) of the dropdown

#### Expected behavior:

Narrator should read the State(Expand/Collapse) of the dropdown

**Comment by khmakoto** [comment]
Hi @Lijing29, this is happening because of a recent change on what Narrator considers advanced information, among which is included the information of the `aria-expanded` attribute. You can check this by going into the codepen you provided, making focus be on the `Dropdown` and then pressing the `[Narrator key] + 0` key combination, which reads advanced information. Upon doing that you should be hearing `collapsed`.

I don't know why the Narrator team decided to change behavior for it to be this way, but there is nothing we can do on the Fabric side to help with this.

**Comment by Lijing29** [comment]
Thanks for the information. If possible, could you please give me the document which mentions that aria-expanded is treated as advanced information by Narrator? I could only find the document showing that Narrator + 0 would read item advanced. But I could not find any document defined what kinds of information are treated as advanced info.
Besides, it seems Narrator + 0 would only announce "Collapsed" when the dropdown is collapsed. and would not announce "Expanded" when it is expanded. Right? I could not find a method to let Narrator announce "Expanded". Do you have any suggestion about this part? Thanks.

**Comment by xugao** [comment]
@Lijing29 - here is an article which gives you some context https://winaero.com/blog/hear-advanced-information-about-controls-narrator-windows-10/. 

Regarding it's not announcing "expanded", that is true. `Narrator + 0` won't give you such info in the expanded case because focus is no longer on the dropdown but on the focused item in the dropdown options when it's expanded. May I ask why in that case you think it's necessary to announce "expanded"  considering the other info it read out seems to provide sufficient knowledge for user?

**Comment by sajsarka** [comment]
@xugao - Issue #11381 is specific to other browsers. Collapsed state is announced in IE and EDGE but not on chrome or Firefox. I tried with `Narrator + 0`, no help. Any workaround how collapsed state can be announced on Chrome/Firefox.
Kindly use the codepen link if needs to reproduce the issue.

**Comment by Lijing29** [comment]
@xugao Thanks for the article. It seems that it is no necessary to announce 'expanded' in my case.

**Comment by xugao** [comment]
@sajsarka - i tried both chrome and FF too. when dropdown is on focus, `Narrator + 0` announces 
"Collapsed". Could you double check?
I am going to close this issue. If you continue to repro #11381, please go ahead reopen it and I will follow up more with you there :)
